templates/HAPROXY_BACKEND_HTTP_HEALTHCHECK_OPTIONS

```diff
@@ -0,0 +1,2 @@
+  option  httpchk GET {healthCheckPath}
```

Comment:
:+1: Sounds reasonable.
@brndnmtthws: Can you move the template changes to the defaults in the script? I think if this is how it's support to be in DCOS by default, it should also be default in the script.